### PR TITLE
List containers should be displayed in sorted order

### PR DESCRIPTION
**Pull request by sangeetha5491**
Thanks for contribution! Please go through following checklist before sending PR.

Links to issue https://github.com/Azure/Azurite/issues/275

### PR Branch Destination

- For Azurite V3, please send PR to `dev` branch.
- For legacy Azurite V2, please send PR to `legacy-dev` branch.

### Always Add Test Cases

Make sure test cases are added to cover the code change.

### Development Guideline

Please go to CONTRIBUTION.md for steps about setting up development environment and recommended Visual Studio Code extensions.

### CODE REVIEW

**Comment by sangeetha5491** [comment]
build

**Comment by sangeetha5491** [comment]
How do I find out why the 4 tests were canceled?

**Comment by sangeetha5491** [comment]
@XiaoningLiu I can't see your comments.

**Comment by blueww** [comment]
@sangeetha5491
1. Please add the change log for the change in: https://github.com/Azure/Azurite/blob/master/ChangeLog.md, you can add it under "Upcoming Release" tag
2. Consider the change for marker  in adding test case.

**Comment by sangeetha5491** [comment]
@XiaoningLiu and @blueww I have addressed the review comments. Please review it again.

File: src/blob/persistence/LokiBlobMetadataStore.ts [hunk 1]
```diff
@@ -293,32 +293,32 @@ export default class LokiBlobMetadataStore
...
+    marker: string = ""
+  ): Promise<[ContainerModel[], string | undefined]> {
     const coll = this.db.getCollection(this.CONTAINERS_COLLECTION);
 
     const query =
       prefix === ""
-        ? { $loki: { $gt: marker }, accountName: account }
+        ? { name: { $gt: marker }, accountName: account }
         : {
             name: { $regex: `^${this.escapeRegex(prefix)}` },
```
Comment (XiaoningLiu) [inline src/blob/persistence/LokiBlobMetadataStore.ts]:
```suggestion
            name: { $regex: `^${this.escapeRegex(prefix)}`, $gt: marker },
```

File: tests/blob/apis/service.test.ts [hunk 2]
```diff
@@ -142,6 +142,104 @@ describe("ServiceAPIs", () => {
...
+    await containerURL3.create(Aborter.none);
+    const result = await serviceURL.listContainersSegment(
+      Aborter.none,
+      containerName2,
+      { maxresults: 1 }
+    );
+    assert.equal(result.containerItems.length, 1);
+    assert.ok(result.containerItems[0].name.startsWith("bb"));
+    assert.equal(result.nextMarker, containerName3);
+    await containerURL1.delete(Aborter.none);
```
Comment (XiaoningLiu) [inline tests/blob/apis/service.test.ts]: Can we use containerName3 as the marker do the query again? Then we will get containerName1 in the response with nextMarker undefined.

Others in the PR looks good.

Comment (sangeetha5491) [reply tests/blob/apis/service.test.ts]: nextMarker contains value of the last container in the previous list. When we fetch the list of containers, it gets fetches the list after the nextMarker. So the response will have nextMarker as containerName1.

Comment (sangeetha5491) [reply tests/blob/apis/service.test.ts]: I'm adding a test case, it looks like it does not set nextMarker to undefined for Loki.

Comment (sangeetha5491) [reply tests/blob/apis/service.test.ts]: done

**Pull request merged**